src/idmap.rs

```diff
@@ -65,13 +65,14 @@ impl Translation for IdTranslation {
 ///
 /// const ASID: usize = 1;
 /// const ROOT_LEVEL: usize = 1;
+/// const NORMAL: Attributes = Attributes::ATTRIBUTE_INDEX_1.union(Attributes::INNER_SHAREABLE);
```

Comment:
So it is the intent that the caller of the API specifies `Attributes::ATTRIBUTE_INDEX_1.union(Attributes::INNER_SHAREABLE)` as the mapping attributes, right? And the 'NORMAL' constant is only for local use? That makes sense.

But I'd still prefer to call this constant NORMAL_CACHEABLE to make it more self_documenting (_normal_ is the least descriptive adjective in the english language)

Reply:
I would expect callers to define their own set of constants for whatever sets of memory attributes they use. This is an example of how they might do that. But good point about the name, I was just using the name we had before. Renamed as you suggested.